packages/cdk/lib/types/engine-options.ts

```diff
@@ -15,6 +16,10 @@ export interface EngineOptions {
    * VPC to run resources in.
    */
   readonly vpc: IVpc;
+  /**
+   * Filesystem provisioned throughput to use for EFS.
+   */
+  readonly iops: Size;
```

Comment:
I think this should be an optional parameter. When the filesystem is S3 you can't supply iops and when the filesystem is EFS then iops is an option.